cmd/commands/root.go

```diff
@@ -77,6 +80,16 @@ func runAudit(auditable ...kubeaudit.Auditable) func(cmd *cobra.Command, args []
 			kubeaudit.WithColor(!rootConfig.noColor),
 		}
 
+		if rootConfig.sarifOut != "" {
+			sarifReport, err := sarif.Create(report)
+
+			if err != nil {
+				log.WithError(err).Fatal("error generating the SARIF output")
```

Comment:
I'll fix this one, because it's a log message and [as per this other comment](https://github.com/Shopify/kubeaudit/pull/453/files#r934696435), can be capitalized